parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -167,6 +167,11 @@ $global:IsAzureCNIOverlayEnabled = {{if IsAzureCNIOverlayFeatureEnabled}}$true{{
 # Kubelet credential provider
 $global:CredentialProviderURL = "{{GetParameter "windowsCredentialProviderURL"}}"
 
+# Kubelet secure TLS bootstrapping settings and related vars
+$global:EnableSecureTLSBootstrapping = [System.Convert]::ToBoolean("{{EnableSecureTLSBootstrapping}}")
+$global:CustomSecureTLSBootstrapAADResource = "{{GetCustomSecureTLSBootstrapAADServerAppID}}"
+$global:SecureTLSBootstrapClientDownloadURL = "https://kubernetesreleases.blob.core.windows.net/aks-tls-bootstrap-client/client-v0.1.0-alpha.4/windows/amd64/tls-bootstrap-client.exe"

```

Comment:
not yet caching

Reply:
We need to use different endpoints per clouds. I think that we need to add a new property for this url and set it in rp

Reply:
yeah, this is also just for testing at the moment basically (upstream staging storage account)

would we need to set it differently for various clouds if we used acs-mirror however?

I think we also want to start putting stuff in MCR from what I've been hearing recently